server/src/server.ts

```diff
@@ -224,13 +230,33 @@ connection.onRequest("powerquery/renameIdentifier", async (params: RenameIdentif
         return await analysis.getRenameEdits(params.newName);
     } catch (error) {
         connection.console.error(
-            `on powerquery/renameIdentifier error ${formatError(ErrorUtils.assertAsError(error))}`,
+            `on powerquery/renameIdentifier error ${ErrorUtils.formatError(ErrorUtils.assertAsError(error))}`,
```

Comment:
I missed the original rename PR but shouldn't this come in as `connector.onRenameRequest` rather than a custom event?

https://github.com/microsoft/vscode-languageserver-node/blob/081ba82a0e49eebec2d1b7d93751290cc1128104/server/src/common/server.ts#L1274

Reply:
Correct. Fixed.